sources/advanced/Cli.ts

```diff
@@ -389,10 +403,13 @@ export class Cli<Context extends BaseContext = BaseContext> implements MiniCli<C
         if (typeof commandClass.usage === `undefined`)
           continue;
 
-        const category = typeof commandClass.usage.category !== `undefined`
-          ? formatMarkdownish(commandClass.usage.category, {format: this.format(colored), paragraphs: false})
+        const category = commandClass.usage.category;
+        const formattedCategory = typeof category !== `undefined`
+          ? formatMarkdownish(category, {format: this.format(colored), paragraphs: false})
           : null;
 
+        formattedCategories.set(category, formattedCategory);
+
```

Comment:
What are those changes for? 🤔

Reply:
I see what you mean; this cache is only being read in one place, so the computation of `formattedCategory` should be moved down to line 458 where it is being used.

Reply:
I somewhat reverted the changes.  I'm not entirely sure what `formatMarkdownish` is doing.  Which is better for `commandsByCategories` keys: the original category, or the markdown-formatted category?